backend/src/node.rs

```diff
@@ -142,12 +142,22 @@ impl Node {
     }
 
     pub fn update_stats(&mut self, interval: &SystemInterval) -> Option<&NodeStats> {
-        if self.stats != interval.stats {
-            self.stats = interval.stats;
-            Some(&self.stats)
-        } else {
-            None
-        }
+      let mut changed = false;
+
+      if let Some(peers) = interval.peers {
+          self.stats.peers = peers;
+          changed = true;
+      }
+      if let Some(txcount) = interval.txcount {
+          self.stats.txcount = txcount;
+          changed = true;
+      }
```

Comment:
Should still check if the value has changed to prevent spamming feeds with unnecessary updates.

```suggestion
      if let Some(peers) = interval.peers {
          if peers != self.stats.peers {
              self.stats.peers = peers;
              changed = true;
          }
      }
      if let Some(txcount) = interval.txcount {
          if txcoint != self.stats.txcount {
              self.stats.txcount = txcount;
              changed = true;
          }
      }
```